servant-foreign/servant-foreign.cabal

```diff
@@ -34,6 +34,7 @@ library
                      , lens       == 4.*
                      , servant    == 0.11.*
                      , text       >= 1.2  && < 1.3
+                     , http-media
```

Comment:
Could you include upper and lower bounds? (It's a mistake that we haven't done this for `http-types`.)